[2021-03-01 20:48:53,068] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.storing_user 2020-05-06T00:00:00+00:00 [queued]>
[2021-03-01 20:48:53,072] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: user_processing.storing_user 2020-05-06T00:00:00+00:00 [queued]>
[2021-03-01 20:48:53,072] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:48:53,072] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-03-01 20:48:53,072] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-03-01 20:48:53,080] {taskinstance.py:1038} INFO - Executing <Task(BashOperator): storing_user> on 2020-05-06T00:00:00+00:00
[2021-03-01 20:48:53,082] {standard_task_runner.py:51} INFO - Started process 251092 to run task
[2021-03-01 20:48:53,085] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'storing_user', '2020-05-06T00:00:00+00:00', '--job-id', '692', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpo_fuq2ci']
[2021-03-01 20:48:53,085] {standard_task_runner.py:76} INFO - Job 692: Subtask storing_user
[2021-03-01 20:48:53,105] {logging_mixin.py:103} INFO - Running <TaskInstance: user_processing.storing_user 2020-05-06T00:00:00+00:00 [running]> on host pop-os.localdomain
[2021-03-01 20:48:53,126] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=storing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-05-06T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-06T00:00:00+00:00
[2021-03-01 20:48:53,127] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-03-01 20:48:53,128] {bash.py:158} INFO - Running command: echo -e ".separator ","
.import /tmp/processed_user.csv users" | sqlite3 /home/i-sip_iot/airflow/airflow.db
[2021-03-01 20:48:53,132] {bash.py:169} INFO - Output:
[2021-03-01 20:48:53,144] {bash.py:177} INFO - Command exited with return code 0
[2021-03-01 20:48:53,155] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=storing_user, execution_date=20200506T000000, start_date=20210302T014853, end_date=20210302T014853
[2021-03-01 20:48:53,167] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-01 20:48:53,176] {local_task_job.py:118} INFO - Task exited with return code 0
